scripts/run_latest_build.sh

```diff
@@ -34,7 +34,7 @@
 #
 
 DOCKER_IP="$(ip addr show docker0 | grep -Po 'inet \K[\d.]+')"
-PUBLIC_IP=${PUBLIC_IP:-$DOCKER_IP}
```

Comment:
It looks like there used to be some hope that ``PUBLIC_IP`` had already been set. Is that still true? Or is it safe to assume that it is never set until this moment, or at least we are happy *always* overwriting any prior value?

Reply:
I followed the getting started docs and saw nothing about setting the PUBLIC_IP. Basically, the mac has to have a default that will be set to fix the bug. If there is some better way, checking if PUBLIC_IP is already set, then checking DOCKER_IP and then a defaulting I wouldn't mind doing that. I would need some direction though for that :)

Reply:
Gotcha, ok. I have no idea if it's important, but just noticed that it was a change in behavior. If you're happy with the behavior, I'm happy. :)

That said, IF it was valuable to use a previously-set PUBLIC_IP, I think you could add a line above this one, and revert this line's change:

```
DOCKER_IP=${DOCKER_IP:-127.0.0.1}
```